### ISSUE TITLE
Get-PASplatform not showing active platforms

### ISSUE THREAD

**Issue by PhilDR007** [pspete/psPAS#359]
**Describe the issue**
After installing version 5.1.16, the command 
_Get-PASplatform -active $true_  
returns the error: 
_CyberArk 11.1.0 does not meet the minimum version requirement of 11.4 for Get-PASPlatform (using ParameterSet: targets)_

Obviously, the command defaults to the "targets" parameterset but I have found no way to force the use of the "platforms" parameterSet, except by using explicitly _-platformID_ but it does not accept wildcards and _-search_ is no longer available

**To Reproduce**
N/A

**Expected behavior**
The list of active platforms should be listed without error. It works using psPAS version 4.0.0.
In other words, based on the CyberArk version it should default to "parameters" and not "targets" parameterSet or provide a switch to force the parameterset for backward compatibility.

**Screenshots & Console Output**
N/A

**Your Environment**

* PowerShell Version:  5.1.14393.2248
* psPAS Version:5.1.16
* CyberArk Version:11.1.0 

**Additional context**
N/A

**Comment by pspete** [comment]
Backward compatibility is always an aim of the module and a consideration for each update and release.
All of the following will work for commands executed against version 11.1: 
```powershell
$ActivePlatforms = Get-PASPlatform -PlatformType Regular -Active $true
#or
$AllPlatforms = Get-PASPlatform -PlatformType Regular
#or
$SomePlatform = Get-PASPlatform -Search oracle
#or
$SinglePlatform = Get-PASPlatform -PlatformID oracle
```
The change in operation is detailed in the project's changelog here: https://github.com/pspete/psPAS/blob/master/CHANGELOG.md#4111-july-14th-2020

However, whilst validating this issue, it was noted that no output was displayed, it appears when the function was updated last year and parametersets were renamed this was not reflected in part of the functions logic used to format the returned data.
The data can still be displayed by piping into `select-object`:

```powershell
$ActivePlatforms | Select-Object *
$AllPlatforms | Select-Object *
$SomePlatform | Select-Object *
$SinglePlatform | Select-Object *
```
The issue with the output will be fixed in the next module release, a related update is already present in the dev branch.
